client/src/App.tsx

```diff
@@ -18,6 +19,9 @@ function App(): JSX.Element {
           <AuthProvider>
             <SocketProvider>
               <Switch>
+                <Route exact path="/profile" component={Profile}>
+                  <Profile />
+                </Route>
```

Comment:
This route would be under `/settings/` in the future, I would consider changing this to reflect that for an easier setup.